### ISSUE TITLE
error in elementor form, and pop up forms

### ISSUE THREAD

**Issue by zeevouwebsite** [elementor/elementor#12960]
## Prerequisites

- [ ] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [ ] The issue still exists against the latest stable version of Elementor.

## Description

on the forms when submit the form it cause this error in console:
Uncaught TypeError: Cannot read property 'error' of undefined
    at onError (common.min.js:2)
    at dispatch (jquery.js:3)
    at r.handle (jquery.js:3)
    at Object.trigger (jquery.js:3)
    at HTMLFormElement.<anonymous> (jquery.js:3)
    at Function.each (jquery.js:2)
    at n.fn.init.each (jquery.js:2)
    at n.fn.init.trigger (jquery.js:3)
    at child.onSuccess (frontend.min.js:2)
    at Object.e.each.r.<computed> [as success] (frontend-modules.min.js:2)
the address of  the file where error occurs is this:
/wp-content/plugins/elementor/assets/js/common.min.js
I think it is part of elementor pro package.
with downgrade it did not fixed.
## Isolating the problem

- [x ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ ] This bug happens with a default WordPress theme active.
- [ ] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
```
					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: MySQL Community Server (GPL) v5.7.31
	PHP Version: 7.4.9
	PHP Max Input Vars: 1000
	PHP Max Post Size: 100M
	GD Installed: Yes
	ZIP Installed: No
	Write Permissions: There are some writing permissions issues with the following directories/files:
		 - WordPress root directory
		 - .htaccess file
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.5.1
	Site URL: https://zeevou.com
	Home URL: https://zeevou.com
	WP Multisite: No
	Max Upload Size: 100 MB
	Memory limit: 256M
	Permalink Structure: /blog/%postname%/
	Language: en-GB
	Timezone: Europe/London
	Debug Mode: Inactive

== Theme ==
	Name: Zeevou
	Version: 1.0.0
	Author: Brainstorm Force
	Child Theme: Yes
	Parent Theme Name: Astra
	Parent Theme Version: 2.6.0
	Parent Theme Author: Brainstorm Force

== User ==
	Role: administrator
	WP Profile lang: en_GB
	User Agent: Mozilla/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/86.0.4240.111 Safari/537.36

== Active Plugins ==
	Akismet Anti-Spam
		Version: 4.1.7
		Author: Automattic

	Analytify - Google Analytics Dashboard
		Version: 3.1.5
		Author: Analytify

	Analytify - Gooogle Analytics Dashboard Widget
		Version: 1.1.4
		Author: Team Analytify

	Astra Pro
		Version: 2.7.0
		Author: Brainstorm Force

	Astra Widgets
		Version: 1.2.7
		Author: Brainstorm Force

	Better Font Awesome
		Version: 1.7.4
		Author: Mickey Kay

	DreamHost Panel Login
		Version: 1.0.0
		Author: DreamHost, danielbachhuber

	Ele Custom Skin
		Version: 3.0.0
		Author: Dudaster.com

	Elementor
		Version: 3.0.12
		Author: Elementor.com

	Elementor Pro
		Version: 3.0.5
		Author: Elementor.com

	Google Language Translator
		Version: 6.0.7
		Author: Translate AI Multilingual Solutions

	Helpie - WordPress KB Wiki Plugin
		Version: 1.9.7
		Author: HelpieWP

	Helpie FAQ (Premium)
		Version: 1.3.1
		Author: HelpieWP

	Horizontal scrolling announcements
		Version: 2.0
		Author: Gopi Ramasamy

	Hummingbird Pro
		Version: 2.6.2
		Author: WPMU DEV

	Intuitive Custom Post Order
		Version: 3.1.2
		Author: hijiri

	Make Column Clickable Elementor
		Version: 1.3.1
		Author: Fernando Acosta

	Pods - Custom Content Types and Fields
		Version: 2.7.22
		Author: Pods Framework Team

	Pods SEO
		Version: 2.2
		Author: Pods Framework Team

	PowerPack Pro for Elementor
		Version: 2.2.0
		Author: Team IdeaBox - PowerPack Elements

	Smush Pro
		Version: 3.7.1
		Author: WPMU DEV

	UpdraftPlus - Backup/Restore
		Version: 1.16.33
		Author: UpdraftPlus.Com, DavidAnderson

	Wordfence Security
		Version: 7.4.12
		Author: Wordfence

	WPMU DEV Dashboard
		Version: 4.10.5
		Author: WPMU DEV

	Yoast Duplicate Post
		Version: 3.2.6
		Author: Enrico Battocchi & Team Yoast

	Yoast SEO Premium
		Version: 15.2
		Author: Team Yoast

== Elements Usage ==
	
	archive : 35
		archive-posts : 4
		button : 8
		form : 1
		heading : 54
		helpie-faq : 6
		helpie-kb-article-listing : 2
		helpie-kb-hero : 1
		html : 3
		icon : 12
		image : 9
		posts : 19
		pp-advanced-tabs : 2
		pp-faq : 12
		pp-info-list : 7
		pp-logo-grid : 2
		pp-video-gallery : 1
		search-form : 2
		sidebar : 2
		spacer : 53
		text-editor : 36
		theme-archive-title : 10
	footer : 1
		button : 3
		form : 1
		heading : 4
		icon-list : 3
		social-icons : 1
		spacer : 2
		text-editor : 3
	header : 3
		breadcrumbs : 4
		html : 10
		image : 8
		nav-menu : 2
		pp-advanced-menu : 2
	loop : 1
		button : 1
		heading : 1
		post-info : 1
		theme-post-featured-image : 1
		theme-post-title : 1
	page : 6
		button : -1
		heading : 3
		helpie-faq : 1
		helpie-kb-article-listing : 2
		helpie-kb-category-listing : 1
		helpie-kb-hero : 1
		image : -4
		posts : 2
		pp-advanced-tabs : 1
		spacer : 7
	popup : 19
		button : 3
		form : 9
		heading : 19
		html : 2
		image : 13
		spacer : 5
		text-editor : 1
	section : 36
		button : 15
		form : 1
		heading : 5
		html : 1
		image : 13
		posts : 21
		pp-advanced-tabs : 1
		pp-posts : 1
		pp-testimonials : 2
		search-form : 1
		shortcode : 1
		spacer : 15
		text-editor : 9
	single : 28
		button : 15
		divider : 4
		heading : 105
		image : 128
		post-comments : 1
		post-info : 1
		shortcode : 80
		sidebar : 1
		spacer : 130
		tabs : 6
		text-editor : 110
		theme-archive-title : 1
		theme-post-content : 1
		theme-post-excerpt : 2
		theme-post-featured-image : 2
		theme-post-title : 3
	widget : 0
		wp-widget-pods_widget_list : 345
	wp-post : 10
		button : 50
		heading : 158
		image : 64
		pp-advanced-tabs : 3
		spacer : 87
		text-editor : 136
		toggle : 5
	wp-page : 28
		button : -28
		countdown : 1
		divider : 3
		form : 5
		global : 1
		heading : 50
		helpie-faq : 1
		html : 1
		image : 5
		image-box : 18
		posts : 1
		pp-advanced-accordion : 1
		pp-info-list : 2
		pp-logo-carousel : 1
		pp-testimonials : 1
		search-form : 1
		shortcode : 3
		spacer : -39
		text-editor : 72
		toggle : 54
	single-page : 2
		heading : 22
		image : 23
		shortcode : 40
		spacer : 12
		text-editor : 4
		theme-post-content : 1
	search-results : 1
		html : 1
		posts : 2

== Log ==
	
JS: showing 20 of 31JS: 2020-10-27 13:01:30 [error X 13][https://zeevou.com/wp-content/plugins/elementor/assets/js/common.min.js:2:128443] Cannot read property \'error\' of undefined 
JS: 2020-10-27 19:38:19 [error X 7][https://zeevou.com/wp-admin/post.php?post=49203&action=elementor#:2:5] Unexpected token \'<\' 
JS: 2020-10-27 19:50:49 [error X 3][https://zeevou.com/wp-admin/post.php?post=49203&action=elementor#:3:31] urlParams.get is not a function 
JS: 2020-10-27 19:58:08 [error X 1][https://zeevou.com/wp-admin/post.php?post=49203&action=elementor#:5:5] href is not defined 
JS: 2020-10-27 19:59:38 [error X 10][https://zeevou.com/wp-admin/post.php?post=49203&action=elementor#:5:8] Unexpected token \'.\' 
JS: 2020-10-28 12:20:47 [error X 1][https://zeevou.com/wp-admin/post.php?post=55872&action=elementor#:14:1] Invalid regular expression: missing / 
JS: 2020-10-28 12:21:16 [error X 2][https://zeevou.com/wp-admin/post.php?post=55872&action=elementor#:15:12] Cannot read property \'addEventListener\' of undefined 
JS: 2020-10-28 12:21:19 [error X 1][https://zeevou.com/wp-admin/post.php?post=55872&action=elementor#:15:10] Failed to execute \'getElementById\' on \'Document\': 1 argument required, but only 0 present. 
JS: 2020-10-28 12:21:21 [error X 3][https://zeevou.com/wp-admin/post.php?post=55872&action=elementor#:15:28] Cannot read property \'addEventListener\' of null 
JS: 2020-10-28 12:22:42 [error X 3][https://zeevou.com/wp-admin/post.php?post=55872&action=elementor#:16:28] Invalid or unexpected token 
JS: 2020-10-28 12:22:47 [error X 11][https://zeevou.com/wp-admin/post.php?post=55872&action=elementor#:17:1] Unexpected token \'}\' 
JS: 2020-10-28 14:19:45 [error X 2][:19:1] o is not defined 
JS: 2020-10-28 14:19:45 [error X 2][:19:1] op is not defined 
JS: 2020-10-28 14:20:47 [error X 2][:19:1] opensection is not defined 
JS: 2020-10-28 14:20:54 [error X 20][https://zeevou.com/wp-admin/post.php?post=55872&action=elementor#:19:15] Unexpected token \'{\' 
JS: 2020-10-28 14:29:51 [error X 2][https://zeevou.com/wp-admin/post.php?post=55872&action=elementor#:23:5] Unexpected token \'var\' 
JS: 2020-10-28 14:30:16 [error X 3][https://zeevou.com/wp-admin/post.php?post=55872&action=elementor#:24:8] Unexpected token \')\' 
JS: 2020-10-28 14:30:31 [error X 1][https://zeevou.com/wp-admin/post.php?post=55872&action=elementor#:26:1] Unexpected end of input 
JS: 2020-10-28 15:43:19 [error X 2][https://zeevou.com/wp-admin/post.php?post=55872&action=elementor#:25:10] Unexpected token \'=\' 
JS: 2020-10-29 10:30:30 [error X 1][https://zeevou.com/free-demo/:13:55] Cannot set property \'autofocus\' of null 

PHP: showing 11 of 11PHP: 2020-10-24 12:30:39 [notice X 6][/home/zeevou/public_html/wp-content/plugins/elementor-pro/modules/dynamic-tags/pods/tags/pods-base.php::35] Undefined index: last_section [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2020-10-24 20:33:27 [notice X 7][/home/zeevou/public_html/wp-content/plugins/elementor/core/common/modules/ajax/module.php::171] Undefined index: data [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2020-10-25 13:26:57 [warning X 2][/home/zeevou/public_html/wp-content/plugins/elementor/core/files/base.php::189] unlink(/home/zeevou/public_html/wp-content/uploads/elementor/css/post-55872.css): Permission denied [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2020-10-25 13:29:02 [warning X 1][/home/zeevou/public_html/wp-content/plugins/elementor/core/files/base.php::180] file_put_contents(/home/zeevou/public_html/wp-content/uploads/elementor/css/post-55872.css): failed to open stream: Permission denied [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2020-10-26 15:02:05 [warning X 5][/home/zeevou/public_html/wp-content/plugins/elementor/core/files/base.php::180] file_put_contents(/home/zeevou/public_html/wp-content/uploads/elementor/css/post-61328.css): failed to open stream: Permission denied [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2020-10-27 20:26:31 [notice X 27][/home/zeevou/public_html/wp-content/plugins/elementor-pro/modules/forms/classes/form-record.php::214] Undefined index: field_type [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2020-10-29 10:07:11 [error X 1][/home/zeevou/public_html/wp-content/plugins/elementor-pro/base/module-base.php::27] Uncaught Error: Class 'ElementorPro\Modules\AnimatedHeadline\Widgets\Animated_Headline' not found in /home/zeevou/public_html/wp-content/plugins/elementor-pro/base/module-base.php:27
Stack trace:
#0 /home/zeevou/public_html/wp-includes/class-wp-hook.php(287): ElementorPro\Base\Module_Base->init_widgets()
#1 /home/zeevou/public_html/wp-includes/class-wp-hook.php(311): WP_Hook->apply_filters()
#2 /home/zeevou/public_html/wp-includes/plugin.php(478): WP_Hook->do_action()
#3 /home/zeevou/public_html/wp-content/plugins/elementor/includes/managers/widgets.php(99): do_action()
#4 /home/zeevou/public_html/wp-content/plugins/elementor/includes/managers/widgets.php(235): Elementor\Widgets_Manager->init_widgets()
#5 /home/zeevou/public_html/wp-content/plugins/elementor/includes/elements/column.php(1032): Elementor\Widgets_Manager->get_widget_types()
#6 /home/zeevou/public_html/wp-content/plugins/elementor/includes/base/element-base.php(916): Elementor\Element_Column->_get_default_child_type()
#7 /home/zeevou/public_html [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2020-10-29 10:07:11 [error X 1][/home/zeevou/public_html/wp-content/plugins/elementor-pro/modules/theme-builder/module.php::28] Uncaught Error: Class 'ElementorPro\Core\Utils' not found in /home/zeevou/public_html/wp-content/plugins/elementor-pro/modules/theme-builder/module.php:28
Stack trace:
#0 /home/zeevou/public_html/wp-content/plugins/elementor-pro/modules/theme-builder/conditions/archive.php(39): ElementorPro\Modules\ThemeBuilder\Module::get_public_post_types()
#1 /home/zeevou/public_html/wp-content/plugins/elementor-pro/modules/theme-builder/conditions/condition-base.php(66): ElementorPro\Modules\ThemeBuilder\Conditions\Archive->register_sub_conditions()
#2 /home/zeevou/public_html/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php(177): ElementorPro\Modules\ThemeBuilder\Conditions\Condition_Base->__construct()
#3 /home/zeevou/public_html/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php(188): ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->register_condition()
#4 /home/zeevou/public_html/wp-content/plugins/elementor-pro/modules/theme-builder/ [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2020-10-29 10:07:11 [error X 1][/home/zeevou/public_html/wp-content/plugins/elementor/core/frontend/render-mode-manager.php::61] Uncaught Exception: 'ElementorPro\Core\App\Modules\SiteEditor\Render_Mode_Template_Preview' must extends 'Render_Mode_Base' in /home/zeevou/public_html/wp-content/plugins/elementor/core/frontend/render-mode-manager.php:61
Stack trace:
#0 /home/zeevou/public_html/wp-content/plugins/elementor-pro/core/app/modules/site-editor/module.php(71): Elementor\Core\Frontend\Render_Mode_Manager->register_render_mode()
#1 /home/zeevou/public_html/wp-includes/class-wp-hook.php(287): ElementorPro\Core\App\Modules\SiteEditor\Module->register_render_mode()
#2 /home/zeevou/public_html/wp-includes/class-wp-hook.php(311): WP_Hook->apply_filters()
#3 /home/zeevou/public_html/wp-includes/plugin.php(478): WP_Hook->do_action()
#4 /home/zeevou/public_html/wp-content/plugins/elementor/core/frontend/render-mode-manager.php(137): do_action()
#5 /home/zeevou/public_html/wp-content/plugins/elementor/includes/frontend.php(198): Elementor\Core\Frontend\Render_Mode_Manager->__construct()
#6 /home/zeevou/public_html/wp-includes/class-wp-hook.p [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2020-10-29 10:07:12 [error X 1][/home/zeevou/public_html/wp-content/plugins/elementor/core/frontend/render-mode-manager.php::61] Uncaught Exception: 'ElementorPro\Modules\Screenshots\Render_Mode_Screenshot' must extends 'Render_Mode_Base' in /home/zeevou/public_html/wp-content/plugins/elementor/core/frontend/render-mode-manager.php:61
Stack trace:
#0 /home/zeevou/public_html/wp-content/plugins/elementor-pro/modules/screenshots/module.php(189): Elementor\Core\Frontend\Render_Mode_Manager->register_render_mode()
#1 /home/zeevou/public_html/wp-includes/class-wp-hook.php(287): ElementorPro\Modules\Screenshots\Module->register_render_mode()
#2 /home/zeevou/public_html/wp-includes/class-wp-hook.php(311): WP_Hook->apply_filters()
#3 /home/zeevou/public_html/wp-includes/plugin.php(478): WP_Hook->do_action()
#4 /home/zeevou/public_html/wp-content/plugins/elementor/core/frontend/render-mode-manager.php(137): do_action()
#5 /home/zeevou/public_html/wp-content/plugins/elementor/includes/frontend.php(198): Elementor\Core\Frontend\Render_Mode_Manager->__construct()
#6 /home/zeevou/public_html/wp-includes/class-wp-hook.php(287): Elementor\Frontend->i [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2020-10-29 10:26:01 [error X 1][/home/zeevou/public_html/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php::177] Uncaught Error: Class '\ElementorPro\Modules\ThemeBuilder\Conditions\General' not found in /home/zeevou/public_html/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php:177
Stack trace:
#0 /home/zeevou/public_html/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php(259): ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->register_condition()
#1 /home/zeevou/public_html/wp-includes/class-wp-hook.php(287): ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->register_conditions()
#2 /home/zeevou/public_html/wp-includes/class-wp-hook.php(311): WP_Hook->apply_filters()
#3 /home/zeevou/public_html/wp-includes/plugin.php(478): WP_Hook->do_action()
#4 /home/zeevou/public_html/wp-settings.php(568): do_action()
#5 /home/zeevou/public_html/wp-config.php(116): require_once('/home/zeevou/pu...')
#6 /home/zeevou/public_html/wp-load.php(37): require_once('/home/zeevou/pu...')
#7 /home/zeevou/public_html/wp-blog-header.php(13): require_on [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

				
</details>

**Comment by Deninichi** [comment]
I have the same issue on at least 4 sites.

**Comment by zeevouwebsite** [comment]
I found that this server error was related to sending email. if you are not sure about problem with sending emails on your site use this plugin:
https://wordpress.org/plugins/check-email/
I have read this article:
https://elementor.com/help/server-errors/
I have configured an SMTP configuration for email with this plugin:
https://wordpress.org/plugins/wp-mail-smtp/
as a result problem with sending emails fixed. and error on the froms fixed too.
